api/src/test/java/no/bibsys/web/DatabaseResourceTest.java

```diff
@@ -403,6 +401,23 @@ public void getEntity_textHtml_entityAsHtml() throws Exception{
         assertThat(html.toLowerCase(), containsString("data-automation-id=\"mylangarray\""));
     }
     
+    @Test
+    public void getEntity_applicationJson_entityAsJson() throws Exception{
+        String registryName = UUID.randomUUID().toString();
+        createRegistry(registryName);
+        EntityDto entity = createEntity(registryName).readEntity(EntityDto.class);
+        
+        Response entityAsJson = getEntityAsJson(registryName, entity.getId());
+        String json = entityAsJson.readEntity(String.class);
+        
+        ObjectMapper mapper = new ObjectMapper();
+        EntityDto readEntity = mapper.readValue(json, EntityDto.class);
+        
+        assertThat(readEntity.getBody(), containsString("label"));

```

Comment:
I guess these should be found in entity::getStringMethod rather than as strings.

Reply:
fixed